src/sentry/api/serializers/models/event.py

```diff
@@ -99,6 +99,11 @@ def serialize(self, obj, attrs, user):
             except TypeError:
                 received = None
 
+        event_type = obj.data.get('type', 'default')
+        metadata = obj.data.get('metadata') or {
+            'title': obj.message_short,
+        }
```

Comment:
Is this value supposed to correspond with the event type `get_metadata` method result as it's generated from `EventManager.save`?